src/api/storage.js

```diff
@@ -5,15 +5,15 @@
 import React from 'react'
 
 /* istanbul ignore file */
-/* eslint-disable no-unused-vars */
 
 export class Storage {
     /**
      * Return an URL for a given media file.
      */
     mediaUrl(name) {
-        throw new Error("not implemented")
+        console.warn('You are using default Storage class.')
+        return name
     }
```

Comment:
_You are using default Storage class_ suggests, that this is an acceptable/expected behaviour. It is not; this class should never be used and was only provided as a superclass (and now as a fallback). This message should be clear that this is a bug and you should fix it.